amp/WEB-INF/src/org/digijava/module/aim/helper/Constants.java

```diff
@@ -197,6 +194,10 @@ private Constants() {
     public static final String ACTIVITY_STATUS_PROPOSED         = "Proposed";
     public static final String ACTIVITY_STATUS_CONSIDERED       = "Considered";
     
+    public static final Set<ApprovalStatus> ACTIVITY_NEEDS_APPROVAL_STATUS_SET = ImmutableSet.of(
+            ApprovalStatus.STARTED_STATUS,
+            ApprovalStatus.EDITED_STATUS,
+            ApprovalStatus.REJECTED_STATUS);
     public static final String ACTIVITY_NEEDS_APPROVAL_STATUS   = "'created', 'started', 'edited', 'rejected'";
```

Comment:
Can we remove this? I saw you replaced it in some places with `ACTIVITY_NEEDS_APPROVAL_STATUS_SET`. On the other hand I see we no longer have 'created' which is present here - could be a leftover.
cc @jdeanquin-dg

Reply:
I forgot to flag this in daily. Actually there are 4 countries that do use 'created' value. If I remove created value from enum those activities will be impossible to read through hibernate.
And actually it is an error that I excluded CREATED from the _SET variable. Adding it back.
From the same category is 'rejected' status which was never used. None of the countries use it. Will report a ticket to tackle both statuses which will probably have to be removed.

Reply:
I made an error and meant 'not_approved' instead of 'rejected'. 'not_approved' is somewhat misleading as it could be represented by 'rejected'/'started'/'edited', basically any status except 'approved' and 'startedapproved'. So we should definitely remove it. On the other hand, 'rejected' is in use by many countries and we'll have to keep it.

@jdeanquin-dg , I am not sure what you mean by approval_status.

The original `ACTIVITY_NEEDS_APPROVAL_STATUS` constant is used for building several queries and it makes sense to keep it as is for now. The set was used for the rest of the cases where we had code like this `if (ACTIVITY_NEEDS_APPROVAL_STATUS.contains("edited")) {...}`.

Reply:
Updated an existing ticket to cleanup approval status. See AMP-26951.

Reply:
sorry for. approval_status I meant created